catkin_tools/verbs/catkin_build/build.py

```diff
@@ -606,8 +610,10 @@ def set_error_state(error_state):
                         total_packages
                     ))
                     # Update status bar
-                    wide_log(msg, rhs=msg_rhs, end='\r')
-                    sys.stdout.flush()
+                    if not status_update_rate or time.time() - last_status_update_time > (1.0 / status_update_rate):
```

Comment:
I would prefer that you check `status_update_rate is not False` since the default value is a boolean whereas the user's provided value is a float. Something like this with a comment describing the condition might be useful:

```python
# if default value (not limited) or enough time has passed, then print a status update
if status_update_rate is False or (time.time() - last_status_update_time) > (1.0 / status_update_rate):
```